pelican/utils.py

```diff
@@ -646,22 +638,37 @@ def process_translations(content_list, order_by=None):
                 for x in lang_items:
                     logger.warning('\t%s', x.source_path)
 
+        # items with `translation` metadata will be used as translations...
+        candidate_items = list(filter(
+            lambda i:
+                i.metadata.get('translation', 'false').lower() == 'false',
+            items))
+        # ...unless all items with that slug are translations
+        if not candidate_items:
+            logger.warning('All times of "%s" are translations', slug)
```

Comment:
"All **items**"? 
Also, I think it's better to mention that we're displaying `slug`. So something like this maybe:
```python
logger.warning('All items with slug "%s" are translations', slug)
```